packages/glob/src/internal-glob-options-helper.ts

```diff
@@ -9,7 +9,8 @@ export function getOptions(copy?: GlobOptions): GlobOptions {
     followSymbolicLinks: true,
     implicitDescendants: true,
     matchDirectories: true,
-    omitBrokenSymbolicLinks: true
+    omitBrokenSymbolicLinks: true,
+    excludeHiddenFiles: false,
```

Comment:
Just to be clear the idea is leave this to default to have hidden but in the `upload-artifact` default that to having this set to `true` unless specifically set in the command?

Reply:
Correct